hilti/toolchain/src/compiler/jit.cc

```diff
@@ -297,8 +297,8 @@ hilti::Result<Nothing> JIT::_compile() {
         if ( auto path = getenv("HILTI_CXX_INCLUDE_DIRS") ) {
             for ( auto&& dir : hilti::rt::split(path, ":") ) {
                 if ( ! dir.empty() ) {
-                    args.emplace_back("-I");
-                    args.emplace_back(dir);
+                    args.insert(args.begin(), std::string(dir));
+                    args.insert(args.begin(), "-I");
```

Comment:
Not added here, but this could be simplified

```suggestion
                    args.insert(args.begin(), {"-I", std::string(dir)});
```